### ISSUE TITLE
Results from counsel-locate aren't colored

### ISSUE THREAD

**Issue by nordlow** [abo-abo/swiper#1201]
Results displayed by counsel-locate are not colored by default. All other matches in minibuffer are. What option(s) control this?

**Comment by abo-abo** [comment]
Indeed, the coloring is not there yet. I'll look into it when I have some time.

**Comment by nordlow** [comment]
Thanks!
